### PR TITLE
Refs [TASK][34331] Create ui player detail

### PR DESCRIPTION
**Pull request by khangvv158**
## Related Tickets
- [#Task 34331: Create ui player detail](https://edu-redmine.sun-asterisk.vn/issues/34331)
## WHAT
- Create ui player detail by fragment dialog
## Evidence (Screenshot or Video)
Screenshot 2021-03-09 190944
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)

### CODE REVIEW

File: app/src/main/res/layout/fragment_player_detail.xml [hunk 1]
```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.playerdetail.PlayerDetailFragment">
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_player_detail.xml:7]: remove

File: app/src/main/res/layout/fragment_player_detail.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
+
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/textViewTitleAge"
+                        style="@style/TextViewTitleStyle"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="Age"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_player_detail.xml]: hard code

File: app/src/main/res/layout/fragment_player_detail.xml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent"
+                        app:layout_constraintVertical_bias="0.09" />
+
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/textViewTitleWeight"
+                        style="@style/TextViewTitleStyle"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="Weight"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_player_detail.xml]: hard code

File: app/src/main/res/layout/fragment_player_detail.xml [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                        app:layout_constraintHorizontal_bias="0.124"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/textViewTitleHeight" />
+
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/textViewTitleHeight"
+                        style="@style/TextViewTitleStyle"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="Height"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_player_detail.xml]: hard code

File: app/src/main/res/layout/fragment_player_detail.xml [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/textViewTitleNationality"
+                        app:layout_constraintVertical_bias="0.27" />
+
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/textViewTitleNationality"
+                        style="@style/TextViewTitleStyle"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="Nationality"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_player_detail.xml]: hard code

File: app/src/main/java/com/sun/uefascore/screen/playerdetail/PlayerDetailFragment.kt [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+            textViewName.text = it.name
+            LoadImageUrl { bitmap ->
+                imageViewAvatar?.setImageBitmap(bitmap)
+            }.execute(it.photo)
+        }
+    }
+
+    companion object {
+
+        private const val PLAYER = "PLAYER"
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/screen/playerdetail/PlayerDetailFragment.kt]: conventions 2.10 
```suggestion
        private const val BUNDLE_PLAYER = "BUNDLE_PLAYER"
```

File: app/src/main/java/com/sun/uefascore/screen/playerdetail/PlayerDetailFragment.kt [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+        fun newInstance(player: Player) = PlayerDetailFragment().apply {
+            arguments = Bundle().apply {
+                putParcelable(PLAYER, player)
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/screen/playerdetail/PlayerDetailFragment.kt]:
```suggestion
            arguments = bundleOf(PLAYER to player)
```

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**